### ISSUE TITLE
terminate all the procs

### ISSUE THREAD

**Issue by XYZ-916** [PaddlePaddle/PaddleDetection#4350]
模型训练途中突然抛异常退出，具体如下：

[10/21 07:50:09] ppdet.utils.checkpoint INFO: Finish loading model weights: /root/.cache/paddle/weights/ResNet50_cos_pretrained.pdparams
loading annotations into memory...
Done (t=0.01s)
creating index...
index created!
[10/21 07:50:31] ppdet.engine INFO: Epoch: [0] [  0/398] learning_rate: 0.001000 loss_rpn_cls: 0.691437 loss_rpn_reg: 0.078807 loss_bbox_cls: 2.366061 loss_bbox_reg: 0.000226 loss: 3.136532 eta: 1 day, 4:31:02 batch_cost: 21.4955 data_cost: 15.0341 ips: 0.1861 images/s
[10/21 07:54:50] ppdet.engine INFO: Epoch: [0] [ 20/398] learning_rate: 0.001080 loss_rpn_cls: 0.558121 loss_rpn_reg: 0.067875 loss_bbox_cls: 0.176637 loss_bbox_reg: 0.000449 loss: 0.754024 eta: 17:40:35 batch_cost: 12.9743 data_cost: 6.5214 ips: 0.3083 images/s
[10/21 07:58:56] ppdet.engine INFO: Epoch: [0] [ 40/398] learning_rate: 0.001160 loss_rpn_cls: 0.172942 loss_rpn_reg: 0.072732 loss_bbox_cls: 0.055649 loss_bbox_reg: 0.000314 loss: 0.321940 eta: 16:53:49 batch_cost: 12.2813 data_cost: 8.7088 ips: 0.3257 images/s
[10/21 08:03:06] ppdet.engine INFO: Epoch: [0] [ 60/398] learning_rate: 0.001240 loss_rpn_cls: 0.097094 loss_rpn_reg: 0.055554 loss_bbox_cls: 0.047764 loss_bbox_reg: 0.002412 loss: 0.204310 eta: 16:41:23 batch_cost: 12.5274 data_cost: 8.9536 ips: 0.3193 images/s
[10/21 08:07:17] ppdet.engine INFO: Epoch: [0] [ 80/398] learning_rate: 0.001320 loss_rpn_cls: 0.056128 loss_rpn_reg: 0.040940 loss_bbox_cls: 0.034346 loss_bbox_reg: 0.002292 loss: 0.138202 eta: 16:33:27 batch_cost: 12.5503 data_cost: 8.8628 ips: 0.3187 images/s
[10/21 08:11:33] ppdet.engine INFO: Epoch: [0] [100/398] learning_rate: 0.001400 loss_rpn_cls: 0.034541 loss_rpn_reg: 0.046760 loss_bbox_cls: 0.034509 loss_bbox_reg: 0.000262 loss: 0.119508 eta: 16:30:21 batch_cost: 12.7661 data_cost: 9.2420 ips: 0.3133 images/s
INFO 2021-10-21 08:13:46,956 launch_utils.py:327] terminate all the procs
ERROR 2021-10-21 08:13:46,956 launch_utils.py:584] ABORT!!! Out of all 3 trainers, the trainer process with rank=[2] was aborted. Please check its log.
INFO 2021-10-21 08:13:49,959 launch_utils.py:327] terminate all the procs

**Comment by nemonameless** [comment]
换训任何检测模型都有这个问题吗？有完整训完过1个epoch的吗？建议发下数据集配置的改动，以及检查下数据集制作过程，也可能是加载到了有问题的数据

**Comment by XYZ-916** [comment]
batch_size调小解决了该问题。
